src/Func.h

```diff
@@ -414,6 +415,11 @@ class Func {
     /** The random seed to use for realizations of this function. */
     uint32_t random_seed;
 
+    /** Whether or not function schedules should be reset before
+     * performing any automatic scheduling. */
+    bool auto_schedule_reset;
```

Comment:
The original motivation for that was two things:
- I didn't know of a way to test if a function's schedule was modified or not. E.g. we could have the auto-scheduler only attempt to auto-schedule functions whose schedules have not been modified by the user.
- For the more powerful heuristic scheduler in the works, I wanted the scheduler to have full control, and I wanted to be sure that the performance numbers I saw were from the heuristic scheduler, not the manual schedule.

But neither of those seems to be a very good reason for it to be in this particular PR, so I could remove it if you think it's best.